README.md

```diff
@@ -1,60 +1,57 @@
-# drbd Cookbook
+drbd Cookbook
```

Comment:
The format changes and removal of the requirements section needs to get reverted.